create-cloud-resources.yml

```diff
@@ -38,6 +38,21 @@
         - username: rattboi
           email: bradon.kanyid@ibm.com
           isadmin: False
+      networks:
+        - name: control-plane
+          cidr: 192.168.10.0/24
+          project: bonnyci
+        - name: nodepool
+          shared: True
+          project: bonnyci
+          cidr: 10.0.0.0/8
+      routers:
+        - name: bonnyci-router
+          network: external
+          project: bonnyci
+          interfaces:
+            - control-plane-subnet
+            - nodepool-subnet
```

Comment:
so if we stop using the project name variables and specify here, we'd need to do that for everything specified in this file. seems like it'd be easier to set as a var, have it used in the roles' tasks so it can DTRT?